testapps/integration/web/index.php

```diff
@@ -29,19 +29,23 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
-
-    $logging = new LoggingClient();
-    $logger = $logging->logger($logName);
-    $logger->write($token);
+    $token = $request->request->get('token');
+    $stderr = fopen('php://stderr', 'w');
+    fwrite($stderr, $token . PHP_EOL);
+    fclose($stderr);
 
     return 'OK';
```

Comment:
It also looks like logging_standard needs to return the name of the error log written to. I'm not sure what it should be in this case. `projects/PROJECT_ID/logs/stderr`?